app/domain/authentication/authn_jwt/identity_provider_factory.rb

```diff
@@ -1,24 +1,35 @@
+require 'command_class'
+
 module Authentication
   module AuthnJwt
     # Factory for jwt identity providers.
     # If Identity variable is configured factory return the decoded_token_provider
     # If the Identity variable is not configured and there is account field in url factory returns url provider
     # If the above conditions are not met exception is raised
-    class IdentityProviderFactory
-      attr_accessor :from_url_provider, :from_decoded_token_provider
-
-      def initialize(authentication_parameters)
-        @from_url_provider = Authentication::AuthnJwt::IdentityFromUrlProvider.new(authentication_parameters)
-        @from_decoded_token_provider = Authentication::AuthnJwt::IdentityFromDecodedTokenProvider.new(authentication_parameters)
+    IdentityProviderFactory = CommandClass.new(dependencies: {
```

Comment:
how about `CreateIdentityProvider`? this fits the command class naming pattern better.

Reply:
also - move `dependencies` to a new line like in other command classes

Reply:
Done